packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -110,6 +110,25 @@ function useStyleWithReset( path, blockName ) {
 	return [ style, setStyle, hasStyle, resetStyle ];
 }
 
+function useFontSizeStyleWithReset(
+	path,
+	blockName,
+	isFluidTypographyEnabled
+) {
+	const [ style, setStyleCallback ] = useStyle( path, blockName );
+	const [ userStyle ] = useStyle( path, blockName, 'user' );
+	const hasStyle = () => !! userStyle;
+	const resetStyle = () => setStyleCallback( undefined );
+	const setStyle = ( newValue, metadata ) => {
+		// Convert font size styles to fluid if fluid is activated.
+		if ( isFluidTypographyEnabled && !! metadata?.slug ) {
+			newValue = `var:preset|font-size|${ metadata?.slug }`;
+		}
+		setStyleCallback( newValue );
+	};
+	return [ style, setStyle, hasStyle, resetStyle ];
```

Comment:
This is a bit of a nit: Just double-checking, but is the purpose of using an array here so that it can be renamed? In the case of `useStyleWithReset` that appears to be the case as it's used many times in the TypographyPanel, but `useFontSizeStyleWithReset` is only used once and is particular to `fontSize` rather than a generic hook to be reused in different cases. So, in this case, would an object return type be slightly safer?

This is definitely not a blocker as this hook isn't exported anywhere, I'm just usually slightly wary of arrays as return types because it's easy to accidentally make a mistake with the positioning in later usage.

Reply:
Yeah, it was a copy and paste job. Thanks for feedback, definitely agree that we can change it to an object.

Cheers!